python/cudf/cudf/_libxx/lib.pxd

```diff
@@ -147,3 +147,15 @@ cdef extern from "<utility>" namespace "std" nogil:
     cdef unique_ptr[column] move(unique_ptr[column])
     cdef unique_ptr[table] move(unique_ptr[table])
     cdef vector[unique_ptr[column]] move(vector[unique_ptr[column]])
+
+cdef extern from "cudf/filling.hpp" namespace "cudf::experimental" nogil:
+    cdef unique_ptr[table] cpp_repeat "cudf::experimental::repeat" (
+        table_view input_table,
+        column_view count,
+        bool check_count,
+    )
+
+    cdef unique_ptr[table] cpp_repeat "cudf::experimental::repeat" (
```

Comment:
```suggestion
    cdef unique_ptr[table] repeat "cudf::experimental::repeat" (
```

Reply:
This is how we have in the developer's guide currently. Currently we have:

```
cudf/
  _libxx/
    lib.pxd  # declarations for cudf functions
    column.pxd  # declarations for Column class
    table.pxd  # declarations for Table class
    *.pyx  # definitions
```

To avoid name clashes, we'd need to have:

```
_libxx
  includes/
    *.pxd  # can declare libcudf functions like `repeat`
    *.pyx  # can define Python function `repeat` without collisions
```

Unfortunately, we can't have:

```
_libxx/
  repeat.pxd  # declares libcudf function `repeat`
  repeat.pyx  # can't define Python `repeat`
              # everything from `repeat.pxd` implicitly imported
```

I'm OK with option 2, and can update the developer guide accordingly.

Reply:
+1 to option 2

Reply:
Oaky - So, this means the new "recommended approach" will be to add a `foo.pxd` to `_libxx/includes/` (rather than adding the foo wrapper definition to `_libxx/lib.pxd`) - Correct?  I do think that will help avoid constant confilicts in `_libxx/lib.pxd` as porting ramps up :)